lib/utilities/monitoringHandler.js

```diff
@@ -3,65 +3,65 @@ const client = require('prom-client');
 
 const collectDefaultMetrics = client.collectDefaultMetrics;
 const numberOfBuckets = new client.Gauge({
-    name: 'cloud_server_number_of_buckets',
+    name: 's3_cloudserver_buckets',
     help: 'Total number of buckets',
 });
 const numberOfObjects = new client.Gauge({
-    name: 'cloud_server_number_of_objects',
+    name: 's3_cloudserver_objects',
     help: 'Total number of objects',
 });
 const lastReportTimestamp = new client.Gauge({
-    name: 'cloud_server_last_report_timestamp',
+    name: 's3_cloudserver_last_report_timestamp',
     help: 'Timestamp of last object/bucket count report',
 });
 const numberOfIngestedObjects = new client.Gauge({
-    name: 'cloud_server_number_of_ingested_objects',
-    help: 'Number of out of band ingestion',
+    name: 's3_cloudserver_ingested_objects',
+    help: 'Total number of out of band objects ingested',
 });
 const dataIngested = new client.Gauge({
-    name: 'cloud_server_data_ingested',
+    name: 's3_cloudserver_ingested_bytes',
     help: 'Cumulative size of data ingested in bytes',
 });
 const dataDiskAvailable = new client.Gauge({
-    name: 'cloud_server_data_disk_available',
+    name: 's3_cloudserver_disk_available_bytes',
     help: 'Available data disk storage in bytes',
 });
 const dataDiskFree = new client.Gauge({
-    name: 'cloud_server_data_disk_free',
+    name: 's3_cloudserver_disk_free_bytes',
     help: 'Free data disk storage in bytes',
 });
 const dataDiskTotal = new client.Gauge({
-    name: 'cloud_server_data_disk_total',
+    name: 's3_cloudserver_disk_bytes',
     help: 'Total data disk storage in bytes',
 });
 
 const labelNames = ['method', 'action', 'code'];
 const httpRequestsTotal = new client.Counter({
     labelNames,
-    name: 'http_requests_total',
-    help: 'Total number of HTTP requests',
+    name: 's3_cloudserver_http_requests_total',
+    help: 'Total number of cloudserver HTTP requests',
 });
 const httpRequestDurationSeconds = new client.Histogram({
-    name: 'http_request_duration_seconds',
+    name: 's3_cloudserver_http_request_duration_seconds',
```

Comment:
This is a standard metrics, that is why a standard name is used.

This is the recommended way to use metrics, to allow "generic" metrics; as discussed with the architects months ago.

c.f. prometheus metrics & label name recommandations: prometheus.io/docs/practices/naming

Reply:
same as above.